lib/configurator/stores/file.js

```diff
@@ -0,0 +1,16 @@
+const assert = require('assert');
+const fs = require('fs');
+
+class FileStore {
+  constructor(config) {
+    assert(config.path, 'FileStore: path required');
+
+    this.path = config.path;
+  }
+
+  fetch(cb) {
+    return fs.readFile(this.path, 'utf-8', cb);
```

Comment:
It is a little bit weird for the store to return an string: Imagine you have a mongo store, would you stringify the result coming from mongo?